tjpcov/main.py

```diff
@@ -486,23 +650,28 @@ def cl_gaussian_cov(self, tracer_comb1=None, tracer_comb2=None,
     #     cov[1423]=None #if want to save memory
         return cov
 
-    def get_all_cov(self, do_xi=False):
+    def get_all_cov(self, do_xi=False, use_nmt=False, **kwards):
         """
         Compute all the covariances and then combine them into one single giant matrix
         Parameters:
         -----------
         two_point_data (sacc obj): sacc object containg two_point data
+        **kwards: The arguments to pass to your chosen covariance estimation
```

Comment:
I think this is typically called `kwargs`

Reply:
Right. Changed